packages/uniforms-antd/src/DateField.js

```diff
@@ -0,0 +1,99 @@
+import connectField     from 'uniforms/connectField';
+import moment           from 'moment';
+import React            from 'react';
+
+import FormGroup        from './FormGroup';
+
+const dateFormat = value => value && value.toISOString().slice(0, -8);
+const dateParse = (timestamp, datestring, onChange, showTime) => {
+    let dateMoment = moment(timestamp);
+    const date = dateMoment.toDate();
+    if (!showTime) {
+        dateMoment = moment(new Date(date.getFullYear(), date.getMonth(), date.getDate(), 0, 0, 0));
+    }
+    onChange(dateMoment.toDate());
+    return;
+};
+
+class Date_ extends (React.Component) {
+    constructor () {
+        super(...arguments);
+        this.state = {
+        };
+    }
+
+    componentWillMount () {
+        if (!(this.props.value instanceof Date) && this.props.value) {
+            const dateMoment = moment(this.props.value,this.props.format);
+            this.props.onChange(dateMoment.toDate());
+        }
+    }
+
+    render () {
+        const {
+            props:{
+                disabled,
+                errorMessage,
+                id,
+                inputRef,
+                label,
+                max,
+                min,
+                name,
+                onChange,
+                placeholder,
+                showInlineError,
+                value,
+                format,
+                allowClear,
+                showTime,
+                info
+              }
+            } = this;
+        const AntD = require('antd');
+        const DatePicker = AntD.DatePicker;
+        const enUS = require('antd/lib/date-picker/locale/en_US');
+        const LocaleProvider = AntD.LocaleProvider;
+        return (
+            <FormGroup errorMessage={errorMessage} id={id} label={label} showInlineError={showInlineError} info={info} >
+                <LocaleProvider locale={enUS}>
```

Comment:
I think `LocaleProvider` should not be put in uniform field, just let developer use `LocaleProvide` to wrap their application.